.pipelines/e2e-windows.yaml

```diff
@@ -72,4 +80,32 @@ stages:
       windowsVMSize: Standard_DS2_v2
       windowsDistro: aks-windows-2022-containerd-gen2
       windowsNodeImageVersion: AKSWindows-2022-containerd-gen2-test-2023.02.07
-      storageAccount: akswinstore2022gen2
\ No newline at end of file
+      storageAccount: akswinstore2022gen2
+
+- stage: test_2022_containerd_gen2_gpu_grid
+  dependsOn: []
+  condition: eq('${{ parameters.test2022containerdgen2gpugrid }}', True)
+  jobs:
+  - template: ./templates/e2e-windows-template.yaml
+    parameters:
+      windowsImage: 2022-containerd-gen2
+      windowsOSSKU: Windows2022
+      windowsVMSize: Standard_NV6ads_A10_v5
```

Comment:
It may be better to set vmsize via variables since we may hit the quota issue.

Reply:
I agree it could hit the quota issue. Is there an existing pattern that we're choosing available vm size dynamically in e2e testing?

Reply:
added pipeline params to choose available vm size. 
todo: remove e2e resource and rerun gpu scenarios to ensure the vm size is. correctly populated from the pipeline params.

Reply:
Seems like you haven't use the pipeline params in this yaml file.